src/explore-education-statistics-admin/src/components/editable/ReorderableAccordion.tsx

```diff
@@ -24,10 +25,11 @@ export interface ReorderableAccordionProps
   extends OmitStrict<AccordionProps, 'openAll'> {
   heading?: string;
   onReorder: (sectionIds: string[]) => void;
+  canUpdateRelease: boolean;
```

Comment:
Would name this more prop more generically i.e. `canReorder`. Where possible, we should think about creating components that can be used in broader contexts than just a single specific use-case :wink: